tests/test_promulgation_seam_560.py

```
@@ -0,0 +1,205 @@
+def test_default_promulgation_stub_passes_every_dossier_without_collaborators():
+    state = object()
+    dossiers = [{"id": 7}, {"id": 11}]
+
+    assert stub_promulgation_verdicts(dossiers, state) == [
+        {"dossier_id": 7, "decision": "promulgated"},
+        {"dossier_id": 11, "decision": "promulgated"},
+    ]
+
+
+def test_injected_promulgation_batch_cannot_silently_omit_a_dossier(read_game):
+    db, state, _content = read_game
```

Ruff (0.16.1)

[warning] Unpacked variable `state` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)